README.md

```diff
@@ -5,29 +5,45 @@ documents that support selective disclosure of claim values.
 
 Written in markdown for the [mmark processor](https://github.com/mmarkdown/mmark).
 
-## Updating Examples
+## Running SD-JWT PoC
 
-All examples in the document are created from [actual running code](demo/simple.py). To run this code, install jwcrypto library, e.g., like so:
+On Debian/Ubuntu systems, you can instead use the packaged version:
```

Comment:
"instead of"...?

Reply:
It's the text that's actually Is in the rradme, moving in another Place needs ti be rephrased. Thelat instead point to the alternative pip installation, i'll fix this, thanks

Reply:
I decided to remove this text and adopt `pip` by default, in setup.py I defined `jwcrypto>=1.3.1`

Reply:
@Sakurann let mw know if this is good to you and I can resovle this conversation or if we may insert again, with a better clarification, that text

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/cbf13bef73911da2f37b969d0cd7b7d1993d667b